models/standardized_models/stripe__line_item_enhanced.sql

```diff
@@ -157,21 +162,25 @@ with invoice_line_item as (
         and invoice.invoice_id = charge.invoice_id
         and invoice.source_relation = charge.source_relation
 
-    left join balance_transaction
-        on charge.balance_transaction_id = balance_transaction.balance_transaction_id
-        and charge.source_relation = balance_transaction.source_relation
+    left join refund
+        on charge.charge_id = refund.charge_id
+        and charge.source_relation = refund.source_relation
+
+    left join balance_transaction bt_charge
+        on charge.balance_transaction_id = bt_charge.balance_transaction_id
+        and charge.source_relation = bt_charge.source_relation
+
+    left join balance_transaction bt_refund
+        on refund.balance_transaction_id = bt_refund.balance_transaction_id
+        and refund.source_relation = bt_refund.source_relation
 
     left join discount 
         on invoice.invoice_id = discount.invoice_id
         and invoice.source_relation = discount.source_relation
 
-    left join refund 
-        on balance_transaction.balance_transaction_id = refund.balance_transaction_id
-        and balance_transaction.source_relation = refund.source_relation
-
     left join account connected_account
-        on balance_transaction.connected_account_id = connected_account.account_id
-        and balance_transaction.source_relation = connected_account.source_relation
+        on bt_charge.connected_account_id = connected_account.account_id
+        and bt_charge.source_relation = connected_account.source_relation
```

Comment:
I am wondering if the adding the following would offer more coverage:

```suggestion
        on coalesce(bt_charge.connected_account_id, bt_refund.connected_account_id)  = connected_account.account_id
        and coalesce(bt_charge.source_relation, bt_refund.source_relation) = connected_account.source_relation
```

Reply:
Wouldn't they be the same value? bt_charge and bt_refund are both from the balance_transaction source, so the connected_account_id values would be the same.

Reply:
This makes sense, went ahead and added.